exp-player/addon/components/exp-card-sort/component.js

```diff
@@ -202,12 +218,14 @@ export default ExpFrameBaseComponent.extend({
       target.unshiftObject(card);
     },
     nextPage() {
+      this.set('cardSortResponse', Ember.copy(this.buckets, true));
```

Comment:
I can't seem to get the `cardSortResponse` computed property to update -- it's watching `'buckets.0.cards.[]', 'buckets.1.cards.[]', 'buckets.2.cards.[]'`